sympy/series/tests/test_series.py

```diff
@@ -1,33 +1,65 @@
 from sympy import sin, cos, exp, E, series, oo, S, Derivative, O, Integral, \
-    Function, PoleError, log, sqrt, N, Symbol, Subs, pi, symbols, atan, LambertW, Rational
+    Function, PoleError, log, sqrt, N, Symbol, Subs, pi, symbols, atan, LambertW, Rational, tan, cot, sec, csc
 from sympy.abc import x, y, n, k
 from sympy.testing.pytest import raises
 from sympy.series.gruntz import calculate_series
 
 
-def test_sin():
+def test_trigonometric():
     e1 = sin(x).series(x, 0)
     e2 = series(sin(x), x, 0)
     assert e1 == e2
 
-
-def test_cos():
     e1 = cos(x).series(x, 0)
     e2 = series(cos(x), x, 0)
     assert e1 == e2
 
+    e1 = tan(x).series(x, 0)
+    e2 = series(tan(x), x, 0)
+    assert e1 == e2
```

Comment:
These tests don't seem very interesting. How exactly do they increase coverage?

I would have thought that a better test would actually show what the series should be.